src/coreclr/interpreter/interpretershared.h

```diff
@@ -11,8 +11,9 @@
 #define INTERP_STACK_SLOT_SIZE 8    // Alignment of each var offset on the interpreter stack
 #define INTERP_STACK_ALIGNMENT 16   // Alignment of interpreter stack at the start of a frame
 
-#define INTERP_METHOD_HANDLE_TAG 4 // Tag of a MethodDesc in the interp method dataItems
+#define INTERP_METHOD_HANDLE_TAG   4 // Tag of a MethodDesc in the interp method dataItems
```

Comment:
Will this work on 32 bit systems like WASM?

Reply:
It was already 4, I just fixed the whitespace. I don't think tagging function pointers is possible in wasm at all unless you use the high bit.
If we're tagging memory addresses, 1 should always work since I'm not aware of any allocator with alignment lower than 4.

So tagging a MethodDesc is valid but using the tag to determine whether a value is a function pointer is not correct. I didn't think of this part until now.